src/main/java/org/sunbird/portal/department/model/AuditEvents.java

```diff
@@ -0,0 +1,148 @@
+package org.sunbird.portal.department.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+"eid",
+"ets",
+"ver",
+"mid",
+"actor",
+"context",
+"object",
+"edata"
+})
+
+public class AuditEvents {
```

Comment:
Need to move this to another new package which should be separate for telemetry related classes.